sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/typedaggregators.scala

```diff
@@ -17,8 +17,10 @@
 
 package org.apache.spark.sql.execution.aggregate
 
+import java.lang
```

Comment:
usually we do `import java.lang.{Long => JLong}`

Reply:
Resolved